inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/feature/ConceptFeatureEditor.java

```diff
@@ -289,6 +291,17 @@ protected void onInitialize() {
     public Component getFocusComponent() {
         return focusComponent;
     }
+    
+    private List<KBHandle> distinctByIri(List<KBHandle> aHandles)
+    {
+        Map<String, KBHandle> hMap = new HashMap<>();
```

Comment:
Potentially changes the order of the list in unpredictive ways. Not sure if this is an issue, but it could be prevented by using a LinkedHashMap().